technical-resources/protocol-overview/smart-contract-addresses-base.md

```diff
@@ -0,0 +1,105 @@
+
+### OpenTermLoan Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| OpenTermLoan | [`0x2d60f1245617813BA230AB32fE89C3B8f53ff71c`](https://basescan.org/address/0x2d60f1245617813BA230AB32fE89C3B8f53ff71c) | [`open-term-loan @ v1.0.1`](https://github.com/maple-labs/open-term-loan/releases/tag/v1.0.1) |
+| OpenTermLoanInitializer | [`0xf7C40F67aA044AaaE180c3f5e836BB23F9DcFd8A`](https://basescan.org/address/0xf7C40F67aA044AaaE180c3f5e836BB23F9DcFd8A) | [`open-term-loan @ v1.0.1`](https://github.com/maple-labs/open-term-loan/releases/tag/v1.0.1) |
+
+### OpenTermLoanManager Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| OpenTermLoanManager | [`0xa6D0b2Ca9F973C81Ef94cCd8c62e4044F22972C1`](https://basescan.org/address/0xa6D0b2Ca9F973C81Ef94cCd8c62e4044F22972C1) | [`open-term-loan-manager @ v1.0.0`](https://github.com/maple-labs/open-term-loan-manager/releases/tag/v1.0.0) |
+| OpenTermLoanManagerInitializer | [`0x5246ADE26673E4F0e1754948b65ceda25EFA2acC`](https://basescan.org/address/0x5246ADE26673E4F0e1754948b65ceda25EFA2acC) | [`open-term-loan-manager @ v1.0.0`](https://github.com/maple-labs/open-term-loan-manager/releases/tag/v1.0.0) |
+
+### PoolManager Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| PoolManager | [`0xad1f6749c09AED1D93cCFAe939045CFEa5011416`](https://basescan.org/address/0xad1f6749c09AED1D93cCFAe939045CFEa5011416) | [`pool-v2 @ v2.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v2.0.0) |
+| PoolManagerInitializer | [`0x5D60901Db2201e812aE438C35F12501D9E8f0996`](https://basescan.org/address/0x5D60901Db2201e812aE438C35F12501D9E8f0996) | [`pool-v2 @ v2.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v2.0.0) |
+
+### WithdrawalManager Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| WithdrawalManager | [`0x5B8387aDa310352198EBFF14a297FAb44428C8CD`](https://basescan.org/address/0x5B8387aDa310352198EBFF14a297FAb44428C8CD) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+| WithdrawalManagerInitializer | [`0x899b7d637e35B2C57E845dB1064a4e58639D3A8D`](https://basescan.org/address/0x899b7d637e35B2C57E845dB1064a4e58639D3A8D) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
+
+## Factory Contracts
+
+| Contract | Address | Commit Hash |
+| -------- | ------- | ----------- |
+| FixedTermLoanFactory | [`0x1915A8dE08A92b846dF7C845e140E4b0714820bd`](https://basescan.org/address/0x1915A8dE08A92b846dF7C845e140E4b0714820bd) | [`fixed-term-loan @ v2.0.0`](https://github.com/maple-labs/fixed-term-loan/releases/tag/v2.0.0) |
```

Comment:
Farhaan should confirm, but I believe the current release is v3.0.1
https://github.com/maple-labs/fixed-term-loan-manager/releases/tag/v3.0.1

Reply:
Appreiciate the confusion here for mainnet we deployed these factories in prior releases but for BASE these factories were deployed with the latest releases from June 2023. 

For FixedTermLoanFactory `fixed-term-loan @ v5.0.1` is the release we deployed 
For FixedTermLoanManagerFactory `fixed-term-loan-manager @ v3.0.1` is the release we deployed
For pool-v2 `pool-v2 @ v2.0.0` is the release we deployed

The other factory versions are correct.

Reply:
(For reference the factorties are identical to the older versions but for accuracy the versions can be updated as listed above)

Reply:
Updated